extensions/amp-twitter/validator-amp-twitter.protoascii

```diff
@@ -30,10 +30,47 @@ tags: {  # <amp-twitter>
   html_format: AMP
   tag_name: "AMP-TWITTER"
   requires_extension: "amp-twitter"
+  # Tweets
   attrs: {
     name: "data-tweetid"
-    mandatory: true
+    mandatory_oneof: "['data-tweetid', 'data-momentid']"
   }
+  attrs: {
+    name: "data-conversation"
+    trigger: {
+      also_requires_attr: "data-tweetid"
+    }
+  }
+  attrs: {
+    name: "data-cards"
+    trigger: {
+      also_requires_attr: "data-tweetid"
+    }
+  }
+  attrs: {
+    name: "data-theme"
+    trigger: {
+      also_requires_attr: "data-tweetid"
+    }
+  }
+  attrs: {
+    name: "data-link-color"
+    trigger: {
+      also_requires_attr: "data-tweetid"
+    }
+  }
+  # Moments
+  attrs: {
+    name: "data-momentid"
```

Comment:
see comment above, let's restrict to numbers.

Reply:
Done.